### ISSUE TITLE
Add profile pictures to object (nwr), history and changeset views

### ISSUE THREAD

**Issue by dieterdreist** [openstreetmap/openstreetmap-website#4403]
### Problem

Add markup for a mini-profile-icon before the username in changeset, history and browse-{nwr} views

### Description

this is a suggestion

### Screenshots

![mini-profile-pics](https://github.com/openstreetmap/openstreetmap-website/assets/581288/e8d7c32f-9f3d-4c01-8368-7625e3844686)

**Comment by AntonKhorev** [comment]
Isn't is too small? The smallest avatar size in use right now is 25x25px. But if you make it larger here, it would look odd as an inline element. So maybe it's better to start with an image. Instead of "Closed over 12 years ago by [image] username" have "[image] username closed over 12 years ago"?

On the other hand maybe there are no avatars on osm data elements to avoid promoting the idea that users "own" them - something that is not done with diary posts for example.

**Comment by dieterdreist** [comment]
You are right that it is quite small, if a bigger image is preferred it does not work well at this inline position. Your suggestion to change the order is good, in alternative the mapper could be named in a new line (similar solution). 

While you don't "own" the elements, you certainly are responsible for (or "own" it if you like) the changeset / edits you uploaded.

**Comment by AntonKhorev** [comment]
Another reason not to have avatars on elements is long history pages. Some elements have 100+ versions. Their history pages are already slow to load. If you add images, that will make things even slower.

**Comment by mmd-osm** [comment]
Agree, inline images don't really work. Maybe this could be shown in a popover window when hovering over the dispaly name. Profile picture and further details are loaded on demand only in that case.

**Comment by HolgerJeromin** [comment]
> Another reason not to have avatars on elements is long history pages. Some elements have 100+ versions. Their history pages are already slow to load. If you add images, that will make things even slower.

We could use [loading="lazy"](https://developer.mozilla.org/en-US/docs/Web/Performance/Lazy_loading#images_and_iframes) like `<img loading="lazy" src="image.jpg" alt="..." />`. That way the browser will not fetch the URL on load.

https://web.dev/articles/browser-level-image-lazy-loading

**Comment by pablobm** [comment]
Found this as I was having the same thought. Small, inline icons like the ones proposed here are common on the web, and often are distinct enough to identify known users at a glance.

Other two places that currently name users without showing an avatar are notes on the map and the user's list of notes (`/user/:username/notes`).
